src/components/Editor/CallSites.js

```diff
@@ -123,21 +123,40 @@ class CallSites extends Component {
 
     const breakpointLines = new Set(breakpoints.map(bp => bp.location.line));
 
-    return callSites.filter(({ location }) =>
-      breakpointLines.has(location.start.line)
+    // Find all callsites that have a breakpoint set on their start line
+    const callSitesForBreakpointLines = callSites
+      .filter(({ location }) => breakpointLines.has(location.start.line))
+      // Filter by only lines with multiple column breakpoints
+      .filter(({ location }, index, arr) => {
+        return (
+          arr.filter(site => site.location.start.line === location.start.line)
+            .length > 1
+        );
+      });
```

Comment:
This block needs to be rewritten.  We've continued to add conditions so a fresh look will do us good.  I think we should index the sites by line, then eliminate those with only one column bp.